main.cs

```diff
@@ -448,12 +449,22 @@ public override void Load()
         GodMode = Config.Bind("Client Options", "GodMode", false);
         AutoRehost = Config.Bind("Client Options", "AutoRehost", false);
 
+        //Proxy
+        UseProxy = Config.Bind("Client Options", "UseProxy", false);
```

Comment:
Client options also need add in `ClientOptionsPatch`

Reply:
ik, but right now im using only for the .cfg file

Reply:
later i will add it